tests/test_files.py

```diff
@@ -225,7 +232,7 @@ def test_create_file(self):
 
 
 class StoreOutputFile(Base):
-    @StoreFiles.return_value
+    @StoreFiles.return_value(cassette=PersistentObjectStorage().cassette)
```

Comment:
Same as above, what does the `return_value` mean here?

(Another UX/naming problem we can solve later..;)